debian/control

```diff
@@ -24,7 +24,7 @@ Build-Depends: debhelper (>= 9.20160709),
 XS-Python-Version: all
 Vcs-Browser: https://github.com/canonical/cloud-init/tree/ubuntu/devel
 Vcs-Git: https://github.com/canonical/cloud-init -b ubuntu/devel
-Standards-Version: 4.4.1
+Standards-Version: 4.5.0
```

Comment:
what do these Version bumps mean for the minimum Debian / Ubuntu version?

Reply:
Hi @igalic, `Standards-Version: 4.5.0` is in Focal already, while dh compat level 13 is available only in Groovy. However this shouldn't matter too much, as the `ubuntu/devel` branch is meant for packaging for the latest in-development release of Ubuntu (Groovy at the moment), while SRUs for older releases are cut from the `ubuntu/<relname>` branches.

Reply:
aaaaahhh
i didn't see your pull request is against that branch